Makefile

```diff
@@ -15,11 +15,10 @@ vendor: glide.yaml
 	# To build without Docker just run "glide install -strip-vendor"
 	docker run --rm \
     -v $(CURDIR):/go/src/github.com/$(PACKAGE_NAME):rw \
-    -v $(HOME)/.glide:/home/user/.glide:rw \
```

Comment:
I think a better approach here would be to have a separate `update-vendor` or similar make target rather than removing the cache from here - but that should be a separate PR.

Could you reinstate the cache and I'll raise a couple of issues covering:  update-vendor and getting this working in a windows environment.